src/pages/transactions/addTransaction.js

```diff
@@ -0,0 +1,102 @@
+import { useRouter } from "next/router";
+
+export default function TransactionForm({ onAddTransaction, onCancel }) {
```

Comment:
Since pages are components in React as well, it's important to maintain consistency between file names and component names. I recommend renaming the file from addTransaction.js to TransactionForm.js to reflect the component name accurately.

Reply:
Fair point, thanks for the hint!